server/get-info-from-googleapis.js

```diff
@@ -71,13 +73,16 @@ const getData = async (rangeName, sheetId) => {
   try {
     const sheetData = await getSheets(sheetId);
 
-    const team = getSheetValues(sheetData.valueRanges, rangeName);
+    cashedSheetData = sheetData;
+
+    const data = getSheetValues(sheetData.valueRanges, rangeName);
 
-    return arrayOfArraysToCollection(team);
+    return arrayOfArraysToCollection(data);
```

Comment:
Why are you caching an intermediate result, not an end one?

Reply:
@epaminond It's my recommendation. I have two reasons why do it:
1) `getData` calls every time when a user goes to `careers` and `team`. That's mean our cache will be refreshed.
2) It'll create more code than now)

Reply:
> getData calls every time when a user goes to careers and team. That's mean our cache will be refreshed.

So what? I suggest smth like this:

```js
const data = getSheetValues(sheetData.valueRanges, rangeName);
cashedSheetData = arrayOfArraysToCollection(data);
return cashedSheetData;
```